src/javascript/app/common/content_visibility.js

```diff
@@ -47,16 +46,26 @@ const mt_company_rule   = 'mtcompany';
 
 const ContentVisibility = (() => {
     const init = () => {
-        if (Client.isLoggedIn()) {
+        const client_is_logged_in = Client.isLoggedIn();
+        if (client_is_logged_in) {
             BinarySocket.wait('authorize', 'landing_company').then(() => {
+                const is_virtual      = State.getResponse('authorize.is_virtual');
+                const landing_company = is_virtual ?
+                    State.getResponse('landing_company.gaming_company.shortcode') ||
+                    State.getResponse('landing_company.financial_company.shortcode') :
+                    State.getResponse('authorize.landing_company_name');
                 controlVisibility(
```

Comment:
perhaps we can turn it to accepting an object, so we can see in an easier way what parameters we are passing to this function? e.g. 
```
controlVisibility({
    is_client_logged_in,
    clients_country                                     : 'default',
    current_landing_company_shortcode: landing_company,
    client_has_mt_company                      : MetaTrader.isEligible(),
    mt5_login_list                                       : State.getResponse('landing_company.mt_financial_company.shortcode'),
});
```

Reply:
unrelated to this PR, but I think it's misleading to have named it `mt5_login_list` when we are sending mt financial shortocde, which seems to be unrelated to login list.